src/QsCompiler/Core/Transformations/SyntaxTreeTransformation.fs

```diff
@@ -73,6 +58,23 @@ type SyntaxTreeTransformation<'T> private (state: 'T, options: TransformationOpt
 
     new(state: 'T) = new SyntaxTreeTransformation<'T>(state, TransformationOptions.Default)
 
+    // These overrides are here since it is impractical to have them live in the definition of CommonTransformationItems.
+
+    override this.OnArgumentTuple argTuple =
+        this.Namespaces.OnArgumentTuple argTuple
+
+    // These overrides are only here to preserve the functionality of the now deprecated methods.
+    // They can be removed and the deprecation warning for this file can be reenabled once the deprecated methods are removed.
+
```

Comment:
It would be great to add a `TODO` here so we don't forget to remove these overrides when the deprecated methods are removed.

Reply:
It should probably be a release `TODO`:
`TODO: RELEASE 2022-09`

Reply:
We cannot really forget to remove them, since when you remove the deprecated methods this code no longer compiles (it calls into the deprecated methods).